packages/util-dynamodb/src/calculateTtl.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Convert the time for an entity to live in the format DynamoDB accepts for
+ * Time To Live (TTL) attributes
+ * https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html
+ * @param {number} hoursToLive Number of hours in the future the time to live should be set for
+ * @return {number} Time in epoch for the TTL
+ */
+export const calculateTtl = (hoursToLive: number): number => {
+  const hours = Number(hoursToLive);
+  console.log(hours);
+  if (isNaN(hours) || !hoursToLive || Array.isArray(hoursToLive)) {
```

Comment:
replace `!hoursToLive` with `hoursToLive <= 0`. The array check is unnecessary since only `hours` is considered after this point.